src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses BioGRID data and maps the interaction types to BEL."""
+
+import logging
+from typing import Iterable, List
+
+import numpy as np
+import pandas as pd
+from protmapper.uniprot_client import get_mnemonic
+from tqdm import tqdm
+
+import pybel.dsl
+from pybel import BELGraph
+
+
+SEP = '\t'
+BIOGRID = 'biogrid'
+SOURCE = 'source'
+TARGET = 'target'
+ALT_SOURCE_ID = 'alt_source_id'
+ALT_TARGET_ID = 'alt_target_id'
+RELATION = 'relation'
+PUBMED_ID = 'pubmed_id'
+UNIPROT = 'uniprot'
+EVIDENCE = 'From BioGRID'
+MODULE_NAME = 'biogrid'
+
+VERSION = '3.5.183'
+BASE_URL = 'https://downloads.thebiogrid.org/Download/BioGRID/Release-Archive'
+URL = f'{BASE_URL}/BIOGRID-{VERSION}/BIOGRID-ALL-{VERSION}.mitab.zip'
+
+#: Relationship types in BioGRID that map to BEL relation 'increases'
+BIOGRID_INCREASES_ACTIONS = {
+    'synthetic genetic interaction defined by inequality',
+    'additive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'decreases'
+BIOGRID_DECREASES_ACTIONS = {
+    'suppressive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'association'
+BIOGRID_ASSOCIATION_ACTIONS = {
+    'direct interaction',
+    'physical association',
+    'colocalization',
+    'association',
+}
+
+BIOGRID_COLUMN_MAPPER = {
+    '#ID Interactor A': SOURCE,
+    'ID Interactor B': TARGET,
+    'Alt IDs Interactor A': ALT_SOURCE_ID,
+    'Alt IDs Interactor B': ALT_TARGET_ID,
+    'Interaction Types': RELATION,
+    'Publication Identifiers': PUBMED_ID,
+}
+
+log = logging.getLogger(__name__)
```

Comment:
rename to logger to avoid confusion with the action of logging `log.log()` would look very strange / this is a community standard

Reply:
Ok, done